ParseLiveQuery/src/main/java/com/parse/OkHttp3SocketClientFactory.java

```diff
@@ -29,7 +29,7 @@ public WebSocketClient createInstance(WebSocketClient.WebSocketClientCallback we
         return new OkHttp3WebSocketClient(mClient, webSocketClientCallback, hostUrl);
     }
 
-    class OkHttp3WebSocketClient implements WebSocketClient {
+    static class OkHttp3WebSocketClient implements WebSocketClient {
```

Comment:
This was otherwise being linked to the outer class at runtime, leading to a possible memory leak.  The Factory and Client do not need to be tied together for any reason, so this should be declared static.